### PR TITLE
add test for "should return autoIncrement primary key"

### PR DESCRIPTION
**Pull request by frogcjn**
add test for "should return autoIncrement primary key"

### CODE REVIEW

File: test/integration/model/create.test.js [hunk 1]
```diff
@@ -1852,6 +1852,33 @@ describe(Support.getTestDialectTeaser('Model'), function() {
...
+          primaryKey: true
+        },
+        createdAt: {
+            field: 'created_at',
+            type: Sequelize.DATE
+        },
+        updatedAt: {
+            field: 'updated_at',
+            type: Sequelize.DATE
+        }
```
Comment (felixfbecker) [inline test/integration/model/create.test.js]: Are all those column definitions needed to reproduce the bug? Or can we omit createdAt/updatedAt and maybe even id?

File: test/integration/model/create.test.js [hunk 2]
```diff
@@ -1852,6 +1852,19 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         expect(m.secret).to.be.eql(M2.secret);
       });
     });
+    it('should return autoIncrement primary key', function() {
+      var Maya = this.sequelize.define('Maya', {
+      });
```
Comment (felixfbecker) [inline test/integration/model/create.test.js]: Put it in one line please :)

**Comment by frogcjn** [comment]
@felixfbecker
It looks the problem not only appeared in MySQL

**Comment by frogcjn** [comment]
It looks like the problem not only appeared on MySQL
@felixfbecker

**Comment by felixfbecker** [comment]
Yes, SQLite has the exact same bug apparently. Can you include a fix for it aswell?

File: test/integration/model/create.test.js [hunk 3]
```diff
@@ -723,6 +723,19 @@ describe(Support.getTestDialectTeaser('Model'), function() {
             });
           });
         });
+        
+        it('should make the autoincremented values available on the returned instances with bulkCreate', function () {
+          var User = this.sequelize.define('user', {});
+
+          return User.sync({force: true}).then(function () {
+            return User.bulkCreate([{}, {}], {returning: true}).then(function (users) {
+              expect(users[0].get('id')).to.be.ok;
```
Comment (felixfbecker) [inline test/integration/model/create.test.js]: You can omit the `to.be.ok` if you check for equality :)

**Comment by felixfbecker** [comment]
The test is not failing anymore

**Comment by frogcjn** [comment]
@felixfbecker So the ok is required.

**Comment by frogcjn** [comment]
@felixfbecker , the test passed because it only test current.dialect.supports.returnValues

so any dialect does not support returnValues will fail the test.

**Comment by felixfbecker** [comment]
LGTM

**Comment by felixfbecker** [comment]
Wasn't this test included in the PR with the fix?

**Comment by felixfbecker** [comment]
Linking https://github.com/sequelize/sequelize/pull/6475

**Pull request merged**